application/lambda/lambda.go

```diff
@@ -194,6 +194,11 @@ func (local *localLambda) writeStaticFile(path string, out io.Writer) error {
 	if err != nil {
 		return err
 	}
+	dir, err := f.Readdir(1)
+	if len(dir) > 0 {
+		f.Close()
+		return local.writeStaticFile( path + "/index.html", out )
+	}
```

Comment:
It feels like `Readdir` here is overkill 😺 

Could you please use something like this

```go
if _, err := os.Stat("/path/to/whatever"); errors.Is(err, os.ErrNotExist) {
  // path/to/whatever does not exist
}
```

Also, for path concatenation it's better to use [filepath.Join](https://pkg.go.dev/path/filepath#Join) - it will take care of few corner cases